libs/internal/src/serialization/json_value.cpp

```diff
@@ -104,11 +104,5 @@ void tag_invoke(boost::json::value_from_tag const&,
     }
 }
 
-tl::expected<Value, JsonError> tag_invoke(
```

Comment:
This wasn't exposed in the json_value header, so I think I introduced it accidentally in a previous change.